### ISSUE TITLE
No support for accented characters

### ISSUE THREAD

**Issue by McGyver91** [dicengine/dice#237]
It seems that DICe does not support a file path with accented characters (à, é, è, etc.). This inhibits the process of generating calculations ( getting the images in a specific file or saving the results)

**Comment by dicengine** [comment]
Yes, this has been a known issue for a while, but we haven't had a chance to fix it yet. Linking in the original issue for reference: https://github.com/dicengine/dice/issues/153

**Comment by dicengine** [comment]
Commit https://github.com/dicengine/dice/commit/c9c5a75740f93c18ed8508d5aa9464b376eb4a91 should have fixed this. This commit is included in the 3.0-beta.4 release https://github.com/dicengine/dice/releases/tag/v3.0-beta.4. See if that works for your unicode filenames.
